src/test/java/com/easypost/Fixture.java

```diff
@@ -118,400 +57,265 @@ public static String uspsCarrierAccountID() {
     }
 
     /**
-     * The pickup service to use.
+     * Get the default tax identifier.
      *
-     * @return USPS `NextDay` service.
+     * @return The default tax identifier
      */
-    public static String pickupService() {
-        return "NextDay";
+    public static HashMap<String, Object> taxIdentifier() {
+        return Objects.requireNonNull(getFixtureData()).taxIdentifiers.basic;
     }
 
     /**
-     * Basic Referral user that is reusable in all test coverage.
+     * Get the default shipment.
      *
-     * @return A map that has the Referral detail info filled.
+     * @return The default shipment
      */
-    public static Map<String, Object> referralUser() {
-        Map<String, Object> referralUser = new HashMap<>();
-
-        referralUser.put("name", "test test");
-        referralUser.put("email", "test@test.com");
-        referralUser.put("phone", "8888888888");
-
-        return referralUser;
+    public static HashMap<String, Object> basicShipment() {
+        return Objects.requireNonNull(getFixtureData()).shipments.basicDomestic;
     }
 
     /**
-     * Report start date for ReportTest.
+     * Get the default full shipment.
      *
-     * @return start date for report.
+     * @return The default full shipment
      */
-    public static String reportDate() {
-        return "2022-04-12";
+    public static HashMap<String, Object> fullShipment() {
+        return Objects.requireNonNull(getFixtureData()).shipments.full;
     }
 
     /**
-     * Report type.
+     * Get the default one-call-buy shipment.
      *
-     * @return report type.
+     * @return The default one-call-buy shipment
      */
-    public static String reportType() {
-        return "shipment";
+    public static HashMap<String, Object> oneCallBuyShipment() {
+        return new HashMap<String, Object>() {{
+            put("to_address", caAddress1());
+            put("from_address", caAddress2());
+            put("parcel", basicParcel());
+            put("service", uspsService());
+            put("carrier_accounts", new ArrayList<String>() {{
+                add(uspsCarrierAccountID());
+            }});
+            put("carrier", usps());
+        }};
     }
 
     /**
-     * Basic tax identifier that is reusable in all test coverage.
+     * Get the default pickup.
      *
-     * @return A map that has the tax identifier info filled.
+     * @return The default pickup
      */
-    public static Map<String, Object> taxIdentifier() {
-        Map<String, Object> taxIdentifier = new HashMap<>();
+    public static HashMap<String, Object> basicPickup() {
+        HashMap<String, Object> fixture = Objects.requireNonNull(getFixtureData()).pickups.basic;
+
+        /*
+        If you need to re-record cassettes, increment the date below and ensure it is one day in the future,
+        USPS only does "next-day" pickups including Saturday but not Sunday or Holidays.
+         */
```

Comment:
Nit: Should really be in the doctoring of the function and not an internal docstring 🤷‍♂️

Reply:
Resolved.